docker-entrypoint.sh

```diff
@@ -46,7 +46,7 @@ if [[ "$1" == "kong" ]]; then
 
     # remove all dangling sockets in $PREFIX dir before starting Kong
     LOGGED_SOCKET_WARNING=0
-    for localfile in "$PREFIX"/*; do
+    for localfile in "$PREFIX"/* "$PREFIX"/sockets/*; do
```

Comment:
> Is the sockets path overwritable?

It is not. It's hardcoded to `{{prefix}}/sockets`.

For a little bit more of my own reasoning on this: I don't know exactly how this repo interacts with Kong/kong with regards to tags and releases. With that in mind, I figured that adding the new path glob instead of just updating the old one (`$PREFIX/*` => `$PREFIX/sockets/*`) and searching _both_ would make this more backwards compatible in case we happen to (re)build any older release trees with the docker entrypoint from this repo.

Reply:
Gotcha, thanks for checking it!